packages/dev/src/main.ts

```diff
@@ -459,6 +460,30 @@ export class NetlifyDev {
 
     this.#cleanupJobs.push(() => runtime.stop())
 
+    // Bootstrap AI Gateway: Fetch AI Gateway tokens and inject them into env
+    if (this.#features.environmentVariables && config?.api && siteID && config?.siteInfo?.url) {
+      await setupAIGateway({
+        api: config.api,
+        env: config.env || {},
+        siteID,
+        siteURL: config.siteInfo.url,
+      })
+
+      // Inject AI_GATEWAY into process.env via runtime
+      if (config.env.AI_GATEWAY) {
+        runtime.env.set('AI_GATEWAY', config.env.AI_GATEWAY.value)
```

Comment:
did we change this to `NETLIFY_AI_GATEWAY` https://docs.netlify.com/build/ai-gateway/overview/

```
NETLIFY_AI_GATEWAY_KEY and NETLIFY_AI_GATEWAY_BASE_URL environment variables are always injected into the AI Gateway-supported runtimes. If you want to mix different setups with your own keys and Netlify’s or you want to be explicit about using AI Gateway keys in your calls, use these env variables as they will never collide with other environment variables values.
```